tldw_Server_API/tests/Collections/test_items_and_outputs_api.py

```
@@ -15,11 +15,26 @@
+    # Override chat_fixtures.preserve_app_state to avoid early app import
+    yield
+
+
+@pytest.fixture(autouse=True)
+def reset_app_overrides():
+    # Override chat_fixtures.reset_app_overrides for these tests
+    yield
+
+
 @pytest.fixture()
 def client_with_user(monkeypatch, tmp_path):
```

Ruff (0.14.4)

Unused function argument: `tmp_path`

(ARG001)